src/server/multi_test.cc

```diff
@@ -756,4 +727,62 @@ TEST_F(MultiTest, TestSquashing) {
   f1.Join();
 }
 
+class MultiEvalTest : public BaseFamilyTest {
+ protected:
+  MultiEvalTest() : BaseFamilyTest() {
+    num_threads_ = kPoolThreadCount;
+    fs_.reset(new absl::FlagSaver);
```

Comment:
Why allocate it dynamically? Can't we just have a member of type `FlagSaver`?
If we can't for some reason (copy c'tor, whatever), please use `make_unique()` instead, as it's a good practice (`reset()` is evil).
And one last thing - why not add this at the `BaseFamilyTest` level for all of Dragonfly's (and Helio) tests?

Reply:
So you are right we dont need to allocate it dynamically. It was how gtest did it so I just took the same approach and did not check why. Anyway now I will change it to not allocate dynamically.
Regarding putting it in BaseFamilyTest, I can do it and our test pass but we have lots of prints from absl from the following type _deps/abseil_cpp-src/absl/flags/internal/flag.cc : 113] RAW: Restore saved value of default_lua_flags to: 
on lots of tests.
where of course the flag name is different for every flag each test changes
I am trying to fix this with setting min sevirity level for absl but this does not work I dont know why.
What do you think ? I dont want to make our test print so many garbage.. 
I am not sure this worth my time trying to fix this if we are going to use the new gtest which will do the flag recover for us

Reply:
I don't see anything wrong with prints, I wouldn't spend much time on trying to eliminate them. BTW I imagine that they will still be there after the gtest upgrade.

Reply:
I disagree with you here. Lots of prints on screen when running test is not good. When running test I just want to see tests output passed/ failed and not some internal logs of the code. I will give it another shot to fix it and if it will not work I will not hold this PR from merge but work on this in a separate PR till figure out how to fix this

Reply:
I was under the impression that in the end of the tests we get a summary, and no logs are printed if the test passes. Perhaps that was only true internally at Google though.